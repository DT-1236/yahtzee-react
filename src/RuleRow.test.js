import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import RuleRow from './RuleRow';

const testRender = (
  <table>
    <tbody>
      <RuleRow />
    </tbody>
  </table>
);

describe('simple smoke tests', () => {
  it('renders without crashing', () => {
    shallow(testRender);
  });

  it('mounts without crashing', () => {
    mount(testRender);
  });
});

describe('simple snapshots', () => {
  it('matches the shallow snapshot', () => {
    const wrapper = shallow(testRender);
    const serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
  });
});
