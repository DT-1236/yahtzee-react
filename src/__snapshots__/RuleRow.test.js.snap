// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple snapshots matches the shallow snapshot 1`] = `
<table>
  <tbody>
    <RuleRow />
  </tbody>
</table>
`;
